src/library/psi_library.jl

```diff
@@ -1149,3 +1149,415 @@ function build_two_zone_5_bus(; kwargs...)
     end
     return sys
 end
+
+const COST_PERTURBATION_NOISE = rand(1_000_000)
```

Comment:
If the idea here is to store a bank of numbers we can use multiple times for reproducibility, it might be cleaner to store a random seed that gets used to re-initialize a RNG each time